File: FreeAds.API/Dtos/ClassifiedAdsForUserUpdate.cs
using System.Collections.Generic;

namespace FreeAds.API.Dtos
{
    public class ClassifiedAdsForUserUpdate
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string City { get; set; }
        public string Category { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        //public ICollection<PhotoForDetailedDto> Photos { get; set; }
    }
}

File: client/src/app/_models/classified-ads-for-user-update.ts
import { PhotoForDetail } from './photo-for-detail';

export interface ClassifiedAdsForUserUpdate {
    id: string;
    title: string;
    description: string;
    city: string;
    category: string;
    email: string;
    phone: string;
    photos: PhotoForDetail[];
    userId: string;
}
